### DISCUSSION TITLE
我的咋是这样

### DISCUSSION THREAD

**Discussion by xintie2023** [share121/inter-knot#1704] [其他]
有没有人知道我的咋是这样？
![Screenshot_20240812_170023_mark via](https://github.com/user-attachments/assets/a671612a-c9c4-4d6b-9c65-0216a170a94d)

**Comment by f4reto** [comment]
没有安装绳网小助手导致的

**Reply by xintie2023** [reply]
咋弄？

**Reply by Inter-Knot-User** [reply]
下载via浏览器，进绳网，点这一行蓝字就行

**Comment by Loxia1121** [comment]
按一下蓝字下载就可以了

**Comment by 147258huan** [comment]
你是咋进来的 点击[http://pinyin.cn/e292777]查看表情

**Comment by ghost** [comment]
via吧, 可以直接安装脚本, 点进去等浏览器弹出安装就行

**Comment by AAAduck** [comment]
把浏览器删了就好了

**Comment by Hanabiiiiii** [comment]
换个域名进试试

**Comment by L-C456** [comment]
点一下安装插件就可以了

**Comment by qingyangprince** [comment]
点蓝字，装插件

**Comment by zhangtianrang** [comment]
这个委托我接下了

**Comment by xxx-zzl** [comment]
那你就安啊

**Comment by SteveNotSet** [comment]
浏览器要安装Tampermonkey才能装这个
而手机Firefox就可以安装Tampermonkey

**Comment by Lbybbs** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/9249bd36125b953ddd445b19151427d6.png" alt="安比 疑惑" width="75" >手机好像不行？
